Updated ingest spreadsheet file source metadata

ESapenaVentura opened the pull request:
As discussed in the demo meeting on Tues 6th, I have added the `file_source` content expected in the supplementary file associated with the Ingest Spreadsheets

ESapenaVentura committed:
Updated ingest spreadsheet file source metadata

ncalvanese1 reviewed (approved):
Looks good to me!

hannes-ucsc committed:
Fix typos and formatting in pre-existing text

hannes-ucsc committed:
Improve formatting of new text

hannes-ucsc commented:
@ESapenaVentura, I added a commit that fixes grammar and formatting in the existing text, and another commit to convert the bullets to a preformatted JSON fragment. Technically, I shouldn't push to other people's branches, so feel free to remove them. 

@ncalvanese1, I should also have asked for a re-review but given that the commits don't alter the semantics, I wanted to save you time.

MightyAx committed:
Specified array in content description example

To meet [file_core schema](https://schema.humancellatlas.org/core/file/6.4.0/file_core)

MightyAx commented:
I've updated the example used to make it **explicit** that `content_description` is an array.
Because we are using [supplementary_file 2.5](https://schema.humancellatlas.org/type/file/2.5.0/supplementary_file) which references [file_core 6.4](https://schema.humancellatlas.org/core/file/6.4.0/file_core), which states that:
`content_description` is an array of [file_content_ontology 1.1](https://schema.humancellatlas.org/module/ontology/1.1.0/file_content_ontology) objects.

hannes-ucsc committed:
Fix one more typo

hannes-ucsc committed:
Reformat JSON
